### PR TITLE
DOC: Document Event Descriptors

### PR DESCRIPTION
**Pull request by danielballan**
What have we wrought?

### CODE REVIEW

File: doc/source/documents.rst [hunk 1]
```diff
@@ -74,23 +74,25 @@ Documents in Detail
...
-The separate times of the individual readings are not thrown away (they are in
-'timestamps') but the overall event 'time' is more often used.
+    {'data':
+        {'temperature': 5.0,
+          'x_setpoint': 3.0,
+          'x_readback': 3.05},
+     'timestamps':
+        {'temperature': 1442521007.9258342,
+         'x_setpoint': 1442521007.5029348,
+         'x_readback': 1442521007.5029348}
```
Comment (tacaswell) [inline doc/source/documents.rst]: missing a closing `}`

Did you mean to drop other3 keys?

File: doc/source/documents.rst [hunk 2]
```diff
@@ -161,4 +161,187 @@ known until the run completes.
...
+typcially change in the middle of a run. A detector's exposure time is usually
+(but not always) in this category.
+
+Devices delinate between the two by providing two different methods that the
+RunEngine can call: ``device.read()`` returns normals readings that are *not*
+considered configuration; ``device.read_configuration()`` returns the readings
+that are considered configuration.
+
+The first time during a run that the RunEngine is told to read a device, it
+reads the device's configuration also. The output of ``device.describe()`` is
```
Comment (tacaswell) [inline doc/source/documents.rst]: should be `describe_configuration()`?

File: doc/source/documents.rst [hunk 3]
```diff
@@ -161,4 +161,187 @@ known until the run completes.
...
+``configuration[device.name]['data']`` and
+``configuration[device.name]['timestamps']``.
+
+In this example, ``x`` has one configuration data key, and ``temp_ctrl``
+happens to provide no configuration information.
+
+.. code-block:: python
+
+    # excerpt of a 'descriptor' document
+    'configuration':
```
Comment (tacaswell) [inline doc/source/documents.rst]: missing opening {

**Comment by danielballan** [comment]
Thanks for the review. Superceded and pushed to master.

**Pull request closed without merging**